### DISCUSSION TITLE
Single user, receiving error about license violation, but I'm the only one here (2024.1.10981)

### DISCUSSION THREAD

**Discussion by jafin** [datalust/seq-tickets#2084] [Help] [answered]
Recently updated to Seq 2024.1.10981
I am a freeloading single user (sorry!) 
First time I have received this warning, ever.
Just to note: I am a single user, there is not even another soul in the house atm. 
I do not have authentication enabled.

What might be the reasons this get triggered?

![image](https://github.com/datalust/seq-tickets/assets/127927/05697608-942b-4e01-bf14-cd632fe7c678)

**Comment by nblumhardt** [comment, marked as answer]
Hi Jason! Thanks for getting in touch.

This is more than likely due to multiple Seq tabs being open, or a clock change on the Seq server; as the message suggests, if you enable authentication the algorithm for detecting misuse changes and you are dramatically less likely to see this.

Hope this helps,
Nick

**Comment by jafin** [comment]
No worries, I've turned on authentication and it's gone away.  Just something I've never experienced in prior versions.. Thanks
